README.md

```diff
@@ -102,8 +102,3 @@ Here's a table:
 | Empty                 | []                    | {"L": []}                                          |
 | Mixed                 | [42, "foo", null]     | {"L": [{"N": "42"}, {"S": "foo"}, {"NULL": true}]} |
 | Duplicates            | ["foo", "bar", "foo"] | {"L": [{"S": "foo"}, {"S": "bar"}, {"S": "foo"}]}  |
```

Comment:
Why delete the whole contributions part rather than just the piece that refers to ES6?